### ISSUE TITLE
Database

### ISSUE THREAD

**Issue by ShreyaV-05** [ShreyaV-05/Waves#7]
- [x] Make the database for the reviews forms
1. Runtime Link: Book review form-http://72.197.230.181:8080/books/bookrev/ Bookstore review form-http://72.197.230.181:8080/books/storerev/
       1. Ticket: https://github.com/ShreyaV-05/Waves/projects/1#card-57815289 
       2. Link to code: 
             Code for model.py (initialize and create db): https://github.com/ShreyaV-05/Waves/blob/a51789d9e457e01882194f0318c59c2fa6b75479/model.py#L40-L56 https://github.com/ShreyaV-05/Waves/blob/a51789d9e457e01882194f0318c59c2fa6b75479/model.py#L140-L176 https://github.com/ShreyaV-05/Waves/blob/a51789d9e457e01882194f0318c59c2fa6b75479/model.py#L71-L86  https://github.com/ShreyaV-05/Waves/blob/a51789d9e457e01882194f0318c59c2fa6b75479/model.py#L187-L228
             Code for app.py(add data to db): https://github.com/ShreyaV-05/Waves/blob/a51789d9e457e01882194f0318c59c2fa6b75479/books/app.py#L45-L61 https://github.com/ShreyaV-05/Waves/blob/a51789d9e457e01882194f0318c59c2fa6b75479/books/app.py#L78-L92
- [x] Make a database for the recommend forms 
1.Link to runtime:Book recommend form-http://72.197.230.181:8080/books/bookrec/   Bookstore recommend form-http://72.197.230.181:8080/books/storerec/
       1. Ticket:https://github.com/ShreyaV-05/Waves/projects/1#card-61733659 
       2. Link to code: 
             Code for model.py (initialize and create db): https://github.com/ShreyaV-05/Waves/blob/a51789d9e457e01882194f0318c59c2fa6b75479/model.py#L23-L39  https://github.com/ShreyaV-05/Waves/blob/a51789d9e457e01882194f0318c59c2fa6b75479/model.py#L99-L139 https://github.com/ShreyaV-05/Waves/blob/a51789d9e457e01882194f0318c59c2fa6b75479/model.py#L57-L70 https://github.com/ShreyaV-05/Waves/blob/a51789d9e457e01882194f0318c59c2fa6b75479/model.py#L178-L185 
             Code for app.py(add data to db): https://github.com/ShreyaV-05/Waves/blob/a51789d9e457e01882194f0318c59c2fa6b75479/books/app.py#L23-L39  https://github.com/ShreyaV-05/Waves/blob/a51789d9e457e01882194f0318c59c2fa6b75479/books/app.py#L63-L76 
- [x] Connect multiple pages from blueprint to the main website
1. Link to runtime: home to see dropdown: http://72.197.230.181:8080/
       1. Ticket:https://github.com/ShreyaV-05/Waves/projects/1#card-61733698
       2. Link to code: 
             Code for home.html: https://github.com/ShreyaV-05/Waves/blob/a51789d9e457e01882194f0318c59c2fa6b75479/templates/home.html#L121-L134

**Comment by ghost** [comment]
I like the idea of being able to recommend books through the system you're looking to create, and if you're able to get the backend with the database working it will be very cool to see the finished product.

**Comment by BillyCherres** [comment]
Good idea with putting the title and the genre of the books into a database. I like how you are sorting based on genre and not based on title.

**Comment by ryanshay18** [comment]
I like the creative structure used by nesting tickets inside of the issue. Great organization structure and specific naming conventions.

**Comment by 1855495** [comment]
Putting the title and genre of the books into a database is a great idea. I like how you're sorting by genre rather than by title.
